packages/mongodb-memory-server-core/src/util/resolve-config.ts

```diff
@@ -15,7 +15,7 @@ export function setDefaultValue(key: string, value: string): void {
 
 let packageJson: Package | undefined;
 export function reInitializePackageJson(directory?: string): void {
-  packageJson = getPackageJson(directory || process.cwd());
```

Comment:
thanks for pointing this out - i thought "just error(because of an empty string), instead of passing a default", changed in 11ff451330a2aeac45f2ff62a12cb440e5eb471e